### PR TITLE
Live page docs update

### PR DESCRIPTION
**Pull request by ChristopherGS**
Please merge after the new SQL editor is turned on by default @petyosi

### CODE REVIEW

File: docs/guides/web-ui/live.md
Comment (ChristopherGS) [inline docs/guides/web-ui/live.md:72]: I've deliberately culled a lot of these - some are no longer relevant, but the section is currently just a bit too much of a "wall of text". I've tried to pull out the key/least intuitive areas to describe

File: docs/guides/web-ui/live.md
Comment (summerscope) [inline docs/guides/web-ui/live.md]:
```suggestion
The live view is the focal point of **Logfire**, where you can see traces arrive in real-time.
```

File: docs/guides/web-ui/live.md
Comment (summerscope) [inline docs/guides/web-ui/live.md]:
```suggestion
To search the live view, click `Search your spans` (keyboard shortcut `/`), this
```

File: docs/guides/web-ui/live.md
Comment (summerscope) [inline docs/guides/web-ui/live.md]:
```suggestion 
opens the search pane:
```

File: docs/guides/web-ui/live.md
Comment (summerscope) [inline docs/guides/web-ui/live.md]:
```suggestion
## SQL search pane
```

File: docs/guides/web-ui/live.md
Comment (summerscope) [inline docs/guides/web-ui/live.md]:
```suggestion
application in real-time (as the name suggests). You can also explore historical data in the **search pane**.
```

**Review by summerscope — CHANGES_REQUESTED** [review]
@ChristopherGS I'm going to finish my review and instead just go ahead and commit my changes to your PR to speed this up 👍

File: docs/guides/web-ui/live.md
Comment (summerscope) [inline docs/guides/web-ui/live.md]:
```suggestion
For confident SQL users, write your queries directly here. For devs who want a bit of help, try the new [PydanticAI](https://ai.pydantic.dev/) feature which generates a SQL query based on your prompt. You can also review the fields available and populate your SQL automatically using the `Reference` list, see more on this below.

**WHERE clause**
As the greyed out `SELECT * FROM RECORDS WHERE` implies, you're searching inside the `WHERE` clause of a SQL query. It has auto-complete & schema hints, so try typing something to get a reminder. To run your query click `Run` or keyboard shortcut `cmd+enter` (or `ctrl+enter` on windows/linux).
```

**Review by summerscope — APPROVED** [review]
ok @ChristopherGS please ignore my ugly commit history - made some edits to this copy to reflect latest changes, I think we will probably want to improve this more at some point but let's not let perfect be the enemy of better. FYI we should hold off on merging this PR until we make the new SQL editor the default mode - @petyosi could you merge this bad boy once that has happened?

**Pull request merged**
